projects/packages/videopress/src/client/admin/components/video-card/types.ts

```diff
@@ -58,6 +58,16 @@ export type VideoDetailsProps = {
 };
 
 export type VideoCardProps = VideoPressVideo & {
+	/**
+	 * Whether the card should look "blank" style.
+	 */
+	isBlank?: boolean;
```

Comment:
Which is the use case for `isBlank`? Don't get where we'll use it

Reply:
It's initial kick to implement something not clearly defined in the design. However, this is a pretty typical case when pulling data from the client. We can keep polishing and tweaking the UI.

Reply:
> Can you please clarify what's that something not clearly defined in the design?

Yes, it's what you mentioned below. "the dimension behavior (and the thumbnail replacement)" is not mentioned explicitly in the design, unless I'm missing something.

Probably _isBlank_ property doesn't make sense, though. Let's keep the card dimension invariable independently whether the data is provided or not.

Reply:
I've removed the `isBlank` property, but the card will keep the dimension even when no data is provided https://github.com/Automattic/jetpack/pull/26124/commits/a1ee7a1841af92fd9b53eb3580c4b06667b39641

Reply:
excellent!

Reply:
thanks both 🙇‍♂️